### PR TITLE
Mon 15847 new plugin pack aws cloud trail

### PR DESCRIPTION
**Pull request by sdepassio**
## Description

Adding documentation for the new AWS CloudTrail monitoring connector

## Target version (i.e. version that this PR changes)

- [ ] 21.10.x (staging)
- [ ] 22.04.x (staging)
- [ ] 22.10.x (staging)
- [ ] 23.04.x (staging)
- [ ] Cloud (staging)
- [x] Monitoring Connectors (staging)
- [ ] 23.10.x (next)

### CODE REVIEW

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md [hunk 1]
```diff
@@ -0,0 +1,413 @@
...
+Configure a service account (access/secret key combo) for which the following privileges have to be granted:
+
+| AWS Privilege                  | Description                                                                               |
+| :----------------------------- |:------------------------------------------------------------------------------------------|
+| cloudtrail:GetTrailStatus | Returns a JSON-formatted list of information about the specified trail.                   |
+| cloudtrail:LookupEvents   | Looks up management events or CloudTrail Insights events that are captured by CloudTrail. |
+
+### Plugin dependencies
+
+To interact with Amazon APIs, you can use either use the *awscli* binary provided by Amazon or *paws*, a Perl AWS SDK (recommended). You must install it on every poller expected to monitor AWS resources.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md:65]:
```suggestion
To interact with the Amazon APIs, you can use either use the *awscli* binary provided by Amazon or *paws*, a Perl AWS SDK (recommended). You must install it on every poller expected to monitor AWS resources.
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md [hunk 2]
```diff
@@ -0,0 +1,413 @@
...
+
+> For now, it is not possible to use *paws* if you are using a proxy to reach AWS Cloudwatch APIs.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md:67]:
```suggestion
> For now, it is not possible to use *paws* if you are using a proxy to reach the AWS Cloudwatch APIs.
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md [hunk 3]
```diff
@@ -0,0 +1,413 @@
...
+
+1. Log into Centreon and add a new host through **Configuration > Hosts**.
+2. Fill the **Name**, **Alias** & **IP Address/DNS** fields according to your ressource settings.
+3. Apply the **Cloud-Aws-CloudTrail-custom** template to the host. A list of macros appears. Macros allow you to define how the connector will connect to the resource, and to customize the connector's behavior.
+4. Fill in the macros you want. Some macros are mandatory.
+
+| Macro         | Description                                                                                           | Default value     | Mandatory   |
+|:--------------|:------------------------------------------------------------------------------------------------------|:------------------|:------------|
+| AWSACCESSKEY  | Set AWS access key                                                                                    |                   |             |
+| AWSASSUMEROLE | Set arn of the role to be assumed                                                                     |                   |             |
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md:188]:
```suggestion
| AWSASSUMEROLE | Set ARN of the role to be assumed                                                                     |                   |             |
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md [hunk 4]
```diff
@@ -0,0 +1,413 @@
...
+| AWSCUSTOMMODE | Choose a custom mode                                                                                  |                   |             |
+| AWSREGION     | Set the region name                                                                                   |                   |             |
+| AWSSECRETKEY  | Set AWS secret key                                                                                    |                   |             |
+| PROXYURL      | Proxy URL if any                                                                                      |                   |             |
+| EXTRAOPTIONS  | Any extra option you may want to add to every command (E.g. a --verbose flag). All options are listed [here](#available-options) |                   |             |
+
+5. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The host appears in the list of hosts, and on page **Resources Status**. The command that is sent by the connector is displayed in the details panel of the host: it shows the values of the macros.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md:195]:
```suggestion
5. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The host appears in the list of hosts, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the host: it shows the values of the macros.
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md [hunk 5]
```diff
@@ -0,0 +1,413 @@
...
+| ERRORMESSAGE  | Filter on an error message pattern                                                                  |                   |             |
+| DELTA         | Time depth for search (minutes)                                                                     |                   |             |
+| WARNINGCOUNT  | Set warning threshold for the number of events                                                      |                   |             |
+| CRITICALCOUNT | Set critical threshold for the number of events                                                     |                   |             |
+| EXTRAOPTIONS  | Any extra option you may want to add to the command (E.g. a --verbose flag). All options are listed [here](#available-options) |                   |             |
+
+</TabItem>
+</Tabs>
+
+3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of service, and on page **Resources Status**. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md:225]:
```suggestion
3. [Deploy the configuration](/docs/monitoring/monitoring-servers/deploying-a-configuration). The service appears in the list of services, and on the **Resources Status** page. The command that is sent by the connector is displayed in the details panel of the service: it shows the values of the macros.
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md [hunk 6]
```diff
@@ -0,0 +1,413 @@
...
+
+## How to check in the CLI that the configuration is OK and what are the main options for?
+
+Once the plugin is installed, log into your Centreon poller's CLI using the
+**centreon-engine** user account (`su - centreon-engine`). Test that the connector 
+is able to monitor an AWS Instance using a command like this one (replace the sample values by yours):
+
+```bash
+/usr/lib/centreon/plugins//centreon_aws_cloudtrail_api.pl \
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md:234]: no default values

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md [hunk 7]
```diff
@@ -0,0 +1,413 @@
...
+| --statistic         | Set cloudwatch statistics (Can be: 'minimum', 'maximum', 'average', 'sum').                       | Paws |
+| --zeroed            | Set metrics value to 0 if none. Usefull when CloudWatch does not return value when not defined.   | Paws |
+| --proxyurl          | Proxy URL if any                                                                                  | Paws |
+
+</TabItem>
+</Tabs>
+
+#### Modes options
+
+All  modes specific options are listed here:
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md:382]:
```suggestion
All modes specific options are listed here:
```

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md [hunk 8]
```diff
@@ -0,0 +1,412 @@
...
+
+</TabItem>
+<TabItem value="paws" label="paws">
+
+| Option              | Description                                                                                       | Type |
+|:--------------------|:--------------------------------------------------------------------------------------------------|:-----|
+| --aws-secret-key    | Set AWS secret key.                                                                               | Paws |
+| --aws-access-key    | Set AWS access key.                                                                               | Paws |
+| --aws-session-token | Set AWS session token.                                                                            | Paws |
+| --aws-role-arn      | Set arn of the role to be assumed.                                                                | Paws |
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudtrail.md:368]:
```suggestion
| --aws-role-arn      | Set ARN of the role to be assumed.                                                                | Paws |
```

**Review by cg-tw — DISMISSED** [review]

**Review by omercier — APPROVED** [review]

**Review by cg-tw — APPROVED** [review]

**Review by omercier — APPROVED** [review]

**Pull request merged**